aws-acm-cert/README.md

```diff
@@ -1,5 +1,7 @@
 # AWS ACM Cert
 
+**_DEPRECATED: Use aws-acm-certificate if using Terraform AWS Provider >3.0._**
```

Comment:
I wonder if we should go ahead and delete this and other deprecated modules now.

Reply:
This specific module is a very large migration, and needs time to go through all the CZI repos. I don't want to freeze the old repos on an old version of cztack, so I'd like aws-acm-cert and aws-acm-certificate to sit side-by-side for now. Other modules, I'm okay with killing.